sentry/src/routes.rs

```diff
@@ -184,8 +215,11 @@
 //!
 //! The route is handled by [`channel::channel_payout()`].
 //!
-//! Request JSON body: [`ChannelPayRequest`](primitives::sentry::ChannelPayRequest)
+//! Request JSON body:
```

Comment:
Please keep the "Request JSON body: [`ChannelPayRequest`](primitives::sentry::ChannelPayRequest)" and and an `Examples` section instead